app/views/nodes/notes/_table.html.erb

```diff
@@ -0,0 +1,47 @@
+<% cache ["node-notes-table", notes.map(&:id), notes.map(&:updated_at).map(&:to_i).sort.last] do %>
```

Comment:
you need to account for the columns in the cache, otherwise in production changing the columns will break the UI.

This is what we have in Issues#index:

```
<% cache ['issues-table', @columns, issues.map(&:id), @issues.map(&:updated_at).map(&:to_i).sort.last] do %>
```